### ISSUE TITLE
Landscape PDF as portrait 

### ISSUE THREAD

**Issue by vpdotta** [DavBfr/dart_pdf#492]
**Describe the bug**
I have a problem while i'm trying to get a landscape PDF. As code below, when i write the file and open, it's on portrait mode and i need to rotate to be a really landscape pdf.

**To Reproduce**
```
 doc.addPage(
      pw.MultiPage(
        orientation: pw.PageOrientation.landscape,
        pageFormat: PdfPageFormat.a4, ...
```

**Expected behavior**
Expect to get a landscape pdf, not a portrait rotated to right

**Flutter Doctor**
`[√] Flutter (Channel dev, 1.24.0-10.2.pre, on Microsoft Windows [versÃ£o 10.0.19041.630], locale pt-BR)
    • Flutter version 1.24.0-10.2.pre at D:\pacotes\flutter
    • Framework revision 022b333a08 (3 weeks ago), 2020-11-18 11:35:09 -0800
    • Engine revision 07c1eed46b
    • Dart version 2.12.0 (build 2.12.0-29.10.beta)

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at C:\Users\lpereira\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Android Studio (version 4.1.0)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] Connected device (1 available)
    • VF550 (mobile) • VF550ANLBETCBB192 • android-arm64 • Android 9 (API 28)`

**Smartphone (please complete the following information):**
VF550 (mobile) • VF550ANLBETCBB192 • android-arm64 • Android 9 (API 28)

**Comment by DavBfr** [comment]
Using layoutPdf, you don't decide how the pdf is printed. The user decides, depending on the printer they have.

**Comment by BenPoell** [comment]
In the PdfPreview it would be nice to already see the preview in landscape. Is this possible or would that be a feature idea?

**Comment by DavBfr** [comment]
@BenPoell this can be a new feature.

**Comment by waketeck** [comment]
Are there any news if that become a feature in future?

**Comment by DavBfr** [comment]
@waketeck it's implemented in printing version 5.0.0
